tests/test_browser.py

```diff
@@ -260,12 +260,36 @@ def make_main(path):
       (absolute_src_path + "@/directory/file.txt", "directory/file.txt"),
       ("some@@file.txt@other.txt", "other.txt"),
       ("some@@file.txt@some@@otherfile.txt", "some@otherfile.txt")]
+    
+    # used for test cases, where .js, .wasm and .data files are in a subdirectory
+    ensure_dir('sub')
+    create_test_file(os.path.join('sub', 'page_sub.html'), '''
+      <!DOCTYPE html>
+      <html lang="en">
+        <head>
+          <script src="sub2/page.js"></script>
+          <script>
+            Module['onRuntimeInitialized'] = function() {
+              //new Module.page();
```

Comment:
is this line needed? (if not, the entire script tag seems unnecessary?)

Reply:
Indeed not necessary, I removed the whole second script tag. Thanks for the suggestion.